awx/ui/client/src/instance-groups/capacity-adjuster/capacity-adjuster.directive.js

```diff
@@ -31,15 +32,24 @@ function CapacityAdjuster (templateUrl) {
                     url: state.url,
                     data
                 };
-                $http(req);
+                $http(req)
+                    .catch(({data, status}) => {
+                        ProcessErrors(data, status, null, {
+                            hdr: 'Error!',
+                            msg: 'Call failed. Return status: ' + status
+                        });
+                    })
+                    .finally(Wait('stop'));
```

Comment:
should this be `() => Wait('stop')` to ensure that it doesn't get called until the promise settles and has reached the finally block?